controllers/fenceagentsremediation_controller_test.go

```diff
@@ -110,29 +113,58 @@ var _ = Describe("FAR Controller", func() {
 		//Scenarios
 		BeforeEach(func() {
 			fenceAgentsPod = buildFarPod()
-			node = getNode(validNodeName)
-			// DeferCleanUp and Create node, fenceAgentsPod and FAR CR
-			DeferCleanup(k8sClient.Delete, context.Background(), node)
+			// DeferCleanUp and Create fenceAgentsPod
+			Expect(k8sClient.Create(context.Background(), fenceAgentsPod)).To(Succeed())
 			DeferCleanup(k8sClient.Delete, context.Background(), fenceAgentsPod)
+		})
+		JustBeforeEach(func() {
+			// DeferCleanUp and Create node, and FAR CR
+			DeferCleanup(k8sClient.Delete, context.Background(), node)
 			DeferCleanup(k8sClient.Delete, context.Background(), underTestFAR)
 			Expect(k8sClient.Create(context.Background(), node)).To(Succeed())
-			Expect(k8sClient.Create(context.Background(), fenceAgentsPod)).To(Succeed())
 			Expect(k8sClient.Create(context.Background(), underTestFAR)).To(Succeed())
 		})
 
-		When("creating FAR CR", func() {
-			It("should build the exec command based on FAR CR", func() {
-				Eventually(func() (bool, error) {
-					res, err := cliCommandsEquality(underTestFAR)
-					return res, err
-				}, 1*time.Second, 500*time.Millisecond).Should(BeTrue())
+		When("creating valid FAR CR", func() {
+			BeforeEach(func() {
+				node = getNode(node01)
+				underTestFAR = getFenceAgentsRemediation(node01, fenceAgentIPMI, testShareParam, testNodeParam)
```

Comment:
This is redundant, see outermost Describe.
And because it's the "default" `node`, maybe move `node = getNode(node01)` to the the outermost Describe as well?